src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -81,4 +81,33 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+    /** Stores the User in Datastore. */
+    public void storeUser(User user) {
+        Entity userEntity = new Entity("User", user.getEmail());
```

Comment:
It's a good idea to use constants for variables like "User", "email", "aboutMe".

For example, I would recommend having something like:

     public class User {
       public static class DatastoreConstants {
          public static String ENTITY_KIND = "User";
          public static String ENTITY_EMAIL_PROPERTY = "email";
          public static String ENTITY_ABOUTME_PROPERTY = "aboutMe";
        }
        // ...
      }